client/src/views/Models/components/Graphs/GraphLegend.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <aside
+    class="graph-legend"
+    :class="{ 'open' : isOpen }"
+    :style="{
+      '--colour-nodes-container': COLOURS.NODES.CONTAINER,
+      '--colour-nodes-default': COLOURS.NODES.DEFAULT,
+      '--colour-labels-dark': COLOURS.LABELS.DARK,
+      '--stroke': DEFAULT_STYLE.node.stroke,
+      '--stroke-width': DEFAULT_STYLE.node.strokeWidth,
+    }"
+  >
+    <header>
+      <button type="button" @click="toggleLegend()">
+        Legend
+        <font-awesome-icon :icon="['fas', toggleIcon ]" />
+      </button>
+    </header>
+    <div class="legend">
+      <ul>
+        <li class="legend-container">
+          <svg viewBox="-5 -5 130 50">
+            <rect x="35" y="0" :rx="DEFAULT_STYLE.node.borderRadius" width="40" height="40"></rect>
+          </svg>
+          Container
+        </li>
+        <li class="legend-parameter">
+          <svg viewBox="-5 -5 130 50">
+            <rect x="40" y="5" :rx="DEFAULT_STYLE.node.borderRadius" width="30" height="30"></rect>
+          </svg>
+          Parameter
+        </li>
+        <li class="legend-initial-condition">
+          <svg viewBox="-5 -5 130 50">
```

Comment:
Maybe we can have this viewBox dimensions as a constant so we don't need to repeat them every time?